packages/frontend/src/components/DashboardTiles/TileForms/LoomTileForm.tsx

```diff
@@ -1,33 +1,25 @@
-import { DashboardChartTile } from '@lightdash/common';
-import React from 'react';
-import { ActionModalProps } from '../../common/modal/ActionModal';
+import { FC } from 'react';
 import Input from '../../ReactHookForm/Input';
 
 export const getLoomId = (value: string): string | undefined => {
     const arr = value.match(/share\/(.*)/);
     return arr?.[1];
 };
 
-const LoomTileForm = ({
-    isDisabled,
-}: Pick<
-    ActionModalProps<DashboardChartTile['properties']>,
-    'useActionModalState' | 'isDisabled'
->) => (
+const LoomTileForm: FC = () => (
```

Comment:
was this intentional ? don't we want to `disable` some inputs ?

Reply:
I think this was coming from copypasting other modals, especially with complex `ActionModalProps` implementations. I did not find the reason to keep disable and the prop was not being used from the outer component.

initially I thought inputs should be disabled on save, but it's not an async action so there is no loading (tile gets created/updated when you save the dashboard, but not from the modal)